### DISCUSSION TITLE
Make sense to still leave plugged in?

### DISCUSSION THREAD

**Discussion by dayaki** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#724] [Q&A]
Thanks for this beautiful piece of software!!!!!

I recently heard, purchased and loving AlDente, but I read somewhere that it is not safe to keep MacBook plugged in for long. So I am curious, is it safe to keep MacBook (M1 Pro) plugged in for long (say 20hrs daily)  while using AlDente? Any short/long term impact on battery?

Thanks

**Comment by eighteentee** [comment]
Looking at activity monitor, if I have Aldente set to stop stop charging at 80%, that is exactly what it does - Activity Monitor reports that the Mac is using AC instead of battery. This makes sense. I can't see any reason why this would cause any issues. I'm in a similar position, except, I take my Mac out twice a week away from power so I always charge up to 100% the night before to get maximum use throughout the day. I have had my Mac since October 2021 and to date my battery health is at 100% with 43 charge cycles. I don't think this is so bad.
